src/main/java/io/confluent/idesidecar/restapi/kafkarest/TopicManagerImpl.java

```diff
@@ -54,7 +56,18 @@ public Uni<TopicDescription> createKafkaTopic(
             clusterId,
             createTopicRequestData.getTopicName(),
             false
-        ));
+            )
+            // The topic may not be immediately available after creation, so we retry a few times
+            // This is also recommended in the Javadoc for UnknownTopicOrPartitionException
+            // "This exception is retryable because the topic or partition might
+            // subsequently be created."
+            .onFailure(UnknownTopicOrPartitionException.class)
+            .retry()
+            // Exponential backoff with a max of 3 retries
+            // Initial delay of 150ms, max delay of 1s
+            .withBackOff(Duration.ofMillis(150), Duration.ofMillis(1000))
```

Comment:
Let's merge this as-is (this is only a test), but in the future I'd do something like to make it more readable:
```suggestion
            .withBackOff(Duration.ofMillis(150), Duration.ofSeconds(1))
```